src/GRANULAR/fix_wall_gran.cpp

```diff
@@ -165,6 +165,9 @@ FixWallGran::FixWallGran(LAMMPS *lmp, int narg, char **arg) :
     wallstyle = REGION;
     idregion = utils::strdup(arg[iarg+1]);
     iarg += 2;
+    // This option is only compatible with fix wall/gran/region
+    if (strcmp(style, "wall/gran/region") != 0)
```

Comment:
```suggestion
    if (!utils::strmatch(style, "^wall/gran/region"))
```
This would be future-proof in case the fix is ported on an accelerator package